accounts/templates/accounts/register.html

```diff
@@ -64,9 +74,11 @@ <h3>Already have an account?</h3>
     //script to add placeholder and classes to fields in the form 
     $(() => {
         let formFields = document.getElementsByTagName("input");
-        formFields[1].placeholder = "Username";
-        formFields[2].placeholder = "Email";
-        formFields[3].placeholder = "Password";
+        formFields[1].placeholder = "First Name";
```

Comment:
I think these placeholders are already in the HTML template so we don't need to add those with Javascript

Reply:
👍 yeah I think youre right